src/reducers/index.js

```diff
@@ -13,8 +13,20 @@ export default (state = initialState, action) => {
   switch (action.type) {
     case INCREMENT:
     // Fill in the body of this case
+      return Object.assign({}, state, {count: state.count + 1});     // does NOT mutate state
+      //  return {count: state.count + 1};     // should do same as above
+
     case DECREMENT:
     // Fill in the body of this case
+      return Object.assign({}, state, {count: state.count - 1});      //
+
+    case MULTIPLY_BY_10:
+      return Object.assign({}, state, {count: state.count * 10});
```

Comment:
Another way to copy the state object is to use the spread operator!

Reply:
I did spread operator at work today with idea that we should not mutate state and that Object.assign is clunky, not sure this was emphasized so much in day 1 vid